app/models/bookmark.rb

```diff
@@ -21,7 +18,7 @@ class Bookmark < ActiveRecord::Base
 
   scope :my_bookmarks, lambda {
     user = User.current
-    return {} unless SETTINGS[:login] and !user.nil?
+    return all unless SETTINGS[:login] and !user.nil?
```

Comment:
Return **all** bookmarks? Seems reasonable if there is no login, but document it (an issue on Redmine would work) Change to user.present?, double negative is confusing.